src/Argu/PreCompute.fs

```diff
@@ -201,6 +206,61 @@ let getHierarchy (uai : UnionArgInfo) =
 
     aux [] uai
 
+module Helpers =
+    /// recognizes and extracts grouped switches
+    /// e.g. -efx --> -e -f -x
+    let groupedSwitchRegex (caseInfo: Lazy<UnionCaseArgInfo[]>) (inheritedParams: Lazy<UnionCaseArgInfo[]>) (helpParam: HelpParam) =
+        lazy(
+            let chars =
+                caseInfo.Value
+                |> Seq.append inheritedParams.Value
+                |> Seq.collect (fun c -> c.CommandLineNames.Value)
+                |> Seq.append helpParam.Flags
+                |> Seq.filter (fun name -> name.Length = 2 && name.[0] = '-' && Char.IsLetterOrDigit name.[1])
+                |> Seq.map (fun name -> name.[1])
+                |> System.Linq.Enumerable.Distinct
```

Comment:
why the Linq distinct function?

Reply:
An artefact of an optimization ;)

The F# version ends up initializing some special comparer (For F# types) that references things in a module from FSharp.Core.
When initialized this module (`fsharp.core!<StartupCode$FSharp-Core>.$Prim-types..cctor()`) loads translation strings for standard exceptions (invalidOp & friends), by searching for dlls on disk and loading them in memory to load resources and this ends up costing ~80ms

In the end i'm not using this function anymore in the "load serialized data" side so I'll change it back (Other functions call into $Prim-types in the main usage, no way to avoid it here)